### DISCUSSION TITLE
ap mode for presentations and users without skills

### DISCUSSION THREAD

**Discussion by honikos** [arendst/Tasmota#15279] [Q&A] [answered]
We are a small association and want to promote solar plug in devises (solar panels with small inverters, plug-in ready). I use "gosund sp 1" in my network as power meter for solar power measuring and it woks perfect.
We want to sponsor some solar plug in panels and want to present the real inverter feed-in into the grid. Some users just want to connect their smartphone direct to the gosund/tasmota device, without integrating it into any w-lan. I read this isn't possible with common tasmota cofiguration. Are there any chances to configure a tasmota - power-plugin als open AP with metering function?  Other ideas for ESP8266 how this could work?

**Comment by Jason2866** [comment, marked as answer]
Not possible without major changes in Tasmota.

**Comment by MacSass** [comment]
This has been discussed elsewhere already and I would love to see an option to keep Tasmota in a Wi-Fi AP mode (eg for using it as garage opener where 8 have no Wi-Fi).
From what I saw it was declined as this would not fit the intended use / focus of Tasmota unfortunately.
